docs/usage/testing.rst

```diff
@@ -61,7 +61,8 @@ We would then test it using the test client like so:
             from my_app.main import app
 
 
-            def test_health_check():
+            @pytest.mark.asyncio
```

Comment:
```suggestion
```
This is not necessary since we are using the "auto" mode of pytest-asyncio

Reply:
Thanks for this, I was wondering why the other tests doesn't are flagged with that line. Sorry